gf-guide/metrics.md

```diff
@@ -134,40 +137,40 @@ Setting vertical metrics usually falls into the following two categories:
 
 Set these values to be the same across all masters to ensure that output instances have equal vertical metrics:
 
--   `typoAscender` and `hheaAscender` set higher than `À`
+-   `typoAscender` and `hheaAscender` set higher than `Ắ`
 -   `typoLineGap` and `hheaLineGap` set to `0`
--   `typoDescender` and `hheaDescender` set lower than the deepest descender of the primary script.
+-   `typoDescender` and `hheaDescender` set to visually leave Caps height centered in the line, but lower than the deepest descender of the primary script.
 -   `winAscent` and `winDecent` set to `yMax` and `yMin` (absolute highest and lowest point in the font)
 -   `use_typo_metrics` is enabled
 
-Expected result: vertical metrics should be around 120-130% of UPM. Anything greater, and the metrics may look too loose.
+Expected result: vertical metrics should be around 130% of UPM. Anything greater, and the metrics may look too loose.
 
 **Example**
 
 A new Latin family has the following qualities:
 
 -   UPM is `1000`
--   `yMax` of tallest `A grave` in the familly (bold for this example) = `940`
--   `yMin` of deepest a-z letter (`g` bold in this family) = `235`
--   Caps height (`H`or `Z` bbox height) = `730`
--   Family's `yMax = 1116` (Bold `A breve hookabove` for this family)
--   Family's `yMin = 320` (ExtraLight `c cedilla` for this family)
+-   `yMax` of tallest `A breve acute` in the familly (bold for this example) = `1000`
```

Comment:
I am afraid the value 1000 may indicate to users that their Abreveacute should be equal to the UPM value. `1010` or else might be clearer to avoid that confusion

Reply:
Changed to 1015 so more evident